ZuluConverter.py

```diff
@@ -21,3 +21,80 @@ def convert_cst_to_zulu(cst_datetime):
 print(convert_cst_to_zulu(datetime.today()))
 
 
+#EST to Zulu
+def convert_est_to_zulu(est_datetime):
+    # Define time zones
+    est_tz = timezone('America/New_York')    # Eastern Standard Time
+    zulu_tz = timezone('UTC')   # Universal Time (Zulu)
+
+    # Localize the input EST datetime
+    localized_est = est_tz.localize(est_datetime)
+
+    # Convert the localized EST datetime to Zulu time
+    zulu_datetime = localized_est.astimezone(zulu_tz)
+
+    zulu_datetime_formatted = zulu_datetime.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+
+    return zulu_datetime_formatted
+
+print(convert_est_to_zulu(datetime.today()))

```

Comment:
every time someone imports this utility module we'll end up with printed statements on standard output stream ... is that really desirable?

if these are meant to be "tests" then I highly recommend they be placed in a separate file, i.e., ZuluConverterTest.py, where, hopefully one day, we have some automation of test execution that can verify code is running as expected